### PR TITLE
[SYCL][Joint Matrix][E2E] Added test for B row_major bfloat16 32x64x1…

### PR DESCRIPTION
**Pull request by YuriPlyakhin**
…6, 1x64x16

### CODE REVIEW

File: sycl/test-e2e/Matrix/joint_matrix_rowmajorA_rowmajorB_impl.hpp [hunk 1]
```diff
@@ -108,42 +107,43 @@ int main() {
...
+                            int32_t>();
+      res += gemm_row_major<8, 16, 32, class su_8x16x32, int8_t, uint8_t,
+                            int32_t>();
+      res += gemm_row_major<8, 16, 32, class uu_8x16x32, uint8_t, uint8_t,
+                            int32_t>();
+
+      if (combination.nsize == 16) { // architecture::intel_gpu_pvc
+        res += gemm_row_major<1, 64, 16, class bf16_1x64x16, bfloat16, bfloat16,
+                              float>();
+        res += gemm_row_major<32, 64, 16, class bf16_32x64x16, bfloat16,
```
Comment (dkhaldi) [inline sycl/test-e2e/Matrix/joint_matrix_rowmajorA_rowmajorB_impl.hpp:128]: Should we add one for 16x16x16 ?

Comment (YuriPlyakhin) [reply sycl/test-e2e/Matrix/joint_matrix_rowmajorA_rowmajorB_impl.hpp:128]: I thought about it, but then I realized that matrix B would be the same as for 8x16x16 combination and decided not to add it, so it doesn't look like we would test more by adding this combination.
At the same time we test here 1x64x16 and 32x64x16, while matrix B is the same for these 2 combinations as well.
So I decided not to spend time on adding 8x16x16 or removing one of 1x64x16 or 32x64x16.
But if you have strong opinion on adding 16x16x16 or removing one of 1x64x16 or 32x64x16 I can do that...

**Review by dkhaldi — APPROVED** [review]
LGTM

**Comment by YuriPlyakhin** [comment]
@intel/llvm-gatekeepers , all green, please, merge

**Pull request merged**